### DISCUSSION TITLE
Conversion of existing Dashboards for Prometheus

### DISCUSSION THREAD

**Discussion by luddite516** [NetApp/harvest#621] [Q&A] [answered]
Our organization is a long-time user of the original Netapp Harvest.  Over the course of several years, we have developed a number of custom dashboards and have heavily modified the original Harvest dashboards.

I am currently evaluating the "new" Harvest with Prometheus.  The metrics specifiers are different from to old Graphite metrics.

Is there any way to import my dashboards from my old system so that they work with Prometheus?  Right now, I am facing a long process of converting every single metric to Prometheus syntax.

**Comment by cgrinds** [comment, marked as answer]
hi @luddite516 I'm not aware of any tool to do that. I know folks have taken the Grafana dashboards and written code to swizzle them since it's json. 

We've done similar kinds of modifications in [grafana.go](https://github.com/NetApp/harvest/blob/main/cmd/tools/grafana/grafana.go) if you want to look for inspiration. We made use of [gjson](https://github.com/tidwall/gjson) and [sjson](https://github.com/tidwall/sjson) but there are so many tools these days to read/write json.
